pkg/operator/ceph/cluster/osd/spec.go

```diff
@@ -399,7 +404,6 @@ func (c *Cluster) makeDeployment(osdProps osdProperties, osd OSDInfo, provisionC
 		{Name: "CEPH_ARGS", Value: "-m $(ROOK_CEPH_MON_HOST)"},
 		blockPathEnvVariable(osd.BlockPath),
 		cvModeEnvVariable(osd.CVMode),
-		dataDeviceClassEnvVar(osd.DeviceClass),
```

Comment:
Why isn't the device class env var passed anymore? It seems unrelated to the osd keyring issue.

Reply:
ops, it was a mistake. reverted